src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -161,12 +162,12 @@ export const ImportAccounts = ({ chainId, onClose }: ImportAccountsProps) => {
   )
   const chainNamespaceDisplayName = asset?.networkName ?? ''
   const [autoFetching, setAutoFetching] = useState(true)
-  const [toggledActiveAccountIds, setToggledActiveAccountIds] = useState<Set<AccountId>>(new Set())
+  const [toggledAccountIds, setToggledAccountIds] = useState<Set<AccountId>>(new Set())
```

Comment:
Ah, but this is a state variable tracking "toggled" account IDs, where they could be previously enabled, disabled, or new. The original implementation tracked enabled state but code review required I implemented it as toggles per #6816 . 

Feels weird yeah, but we have to track it this was to avoid having 2 paradigms of state tracking running at the same time.